### PR TITLE
Issue 8 fix enabling default accessibility config

### PR DESCRIPTION
**Pull request by pragsanshi**
Resolving https://github.com/intuit/proof/issues/8 by hard-setting the default configs to true. Alternatively, I can also just clean up those false settings if the default is true. Please let me know which is preferred, thanks!

### CODE REVIEW

File: plugins/accessibility/src/main.ts [hunk 1]
```diff
@@ -18,15 +18,15 @@ const defaultAxeConfig: axe.Spec = {
   rules: [
     {
       id: 'color-contrast',
-      enabled: false
+      enabled: true
```
Comment (tylerkrupicka) [inline plugins/accessibility/src/main.ts]: Hey @pragsanshi! I think in this case we want to just make the default configuration an empty object (`{}`). That will turn on all Axe rules by default, which is probably what users of this plugin will expect.

Comment (tylerkrupicka) [reply plugins/accessibility/src/main.ts]: Thanks for contributing!

**Comment by pragsanshi** [comment]
@tylerkrupicka thanks for the feedback, makes sense and cleaned it up in the latest commit

**Comment by adierkens** [comment]
:rocket: PR was released in v0.0.8 :rocket:

**Pull request merged**